osu.Game.Rulesets.Osu/Objects/Slider.cs

```diff
@@ -248,6 +248,8 @@ private void updateNestedPositions()
 
             if (TailCircle != null)
                 TailCircle.Position = EndPosition;
+
+            // Positions of other nested hitobjects are not updated
```

Comment:
This is the type of comment that just really makes me question "why aren't we doing that though?"

I'm not sure anyone knows the answer to that but does that do anything to fix the issue? Why was this comment added here?